.circleci/config.yml

```diff
@@ -243,7 +252,6 @@ workflows:
                 - /^Version-v(\d+)[.](\d+)[.](\d+)/
           requires:
             - prep-build
-            - get-changed-files-with-git-diff
```

Comment:
this is not needed here, since the vault decrypt only runs on develop and RC branches